homework_1/task_02/taxes/testCases_test.go

```diff
@@ -0,0 +1,149 @@
+package taxes_test
+
+import (
+	"code-cadets-2021/homework_1/task_02/taxes"
+	"math"
+)
+
+type testCase struct {
+	inputAmount float64
+	class       []taxes.TaxClass
+	outputTax   float64
+	error       bool
+}
+
+func getTestCases() []testCase {
+	return []testCase{
+		{
+			inputAmount: 7000,
+			class: []taxes.TaxClass{
+				{UpperThreshold: 1000, Percentage: 0},
+				{UpperThreshold: 5000, Percentage: 0.10},
+				{UpperThreshold: 10000, Percentage: 0.20},
+				{UpperThreshold: math.Inf(1), Percentage: 0.30},
```

Comment:
Would this be nicer in the terms of formatting?

```suggestion
				{
					UpperThreshold: 1000,
					Percentage: 0,
				},
				{
					UpperThreshold: 5000,
					Percentage: 0.10,
				},
				{
					UpperThreshold: 10000,
					Percentage: 0.20,
				},
				{
					UpperThreshold: math.Inf(1),
					Percentage: 0.30,
				},
```